_sass/components/_library-card.scss

```diff
@@ -0,0 +1,78 @@
+// Library card ------------- //
+
+// Sets a height on library entries so they're the same height and don't break the grid
+.library-card {
+  box-shadow: 0 0 0 1px $color-gray-lighter;
+  display: block;
+  margin-bottom: 3rem;
+  padding-bottom: 3rem;
+  position: relative;
+  text-decoration: none;
+
+  @include media($medium-screen) {
+    display: flex;
+    flex-direction: column;
+    margin-right: 2.3%;
+    min-height: 350px;
```

Comment:
remove the `min height`  and add a new style for `library-container` and set the flex properties on that.

```
.library-container {
display: flex;
flex-direction: row;
flex-wrap: wrap;
}
```
This will allow each row to be as tall as the tallest content and adapt to the content 👍 
See the screenshot example. 
![screen shot 2017-07-25 at 12 55 32 pm](https://user-images.githubusercontent.com/1449852/28583828-902f3014-7138-11e7-8601-824e4148a083.png)